tests/unit/src/unit/issues/Issue6195.hx

```diff
@@ -0,0 +1,11 @@
+package unit.issues;
+
+class Issue6195 extends Test {
+	static var foo(get, default):Int;
+	static function get_foo() return foo;
```

Comment:
Maybe make the getter do something like `+ 1`, then test set + get the variable (and verify that it's going through the getter)?

Reply:
Yep, it needs better tests. I really should take some time to improve my unit tests workflow..

I'm also interested in cases where this would bring up problems that current tests wouldn't catch. Can't think of one atm =/

Reply:
There shouldn't really be such cases, since this is basically a "new" feature.